src/cmake/common-settings.cmake

```diff
@@ -220,14 +213,3 @@ macro(executable_strip TARGET)
 		endif()
 	endif()
 endmacro()
-
-if("${CMAKE_BUILD_TYPE}" STREQUAL "Release")
```

Comment:
This part was unnecessary: cmake already appends the config-specific flags to common flags, see [doc](https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_FLAGS_CONFIG.html#variable:CMAKE_%3CLANG%3E_FLAGS_%3CCONFIG%3E)